<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmyWidget</class>
 <widget class="QWidget" name="QmyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmyWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设置男生年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoy"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnBoyInc">
        <property name="text">
         <string>boy长大一岁</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClassInfo">
        <property name="text">
         <string>类的元对象信息</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>设置女生年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinGirl"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnGirlInc">
        <property name="text">
         <string>girl长大一岁</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空文本框</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
